pkg/login/social/generic_oauth.go

```diff
@@ -412,28 +414,21 @@ func (s *SocialGenericOAuth) FetchPrivateEmail(client *http.Client) (string, err
 	return email, nil
 }
 
-func (s *SocialGenericOAuth) FetchTeamMemberships(client *http.Client) ([]int, bool) {
-	type Record struct {
-		Id int `json:"id"`
-	}
-
-	response, err := s.httpGet(client, fmt.Sprintf(s.apiUrl+"/teams"))
+func (s *SocialGenericOAuth) FetchTeamMemberships(client *http.Client) ([]string, bool) {
+	response, err := s.httpGet(client, fmt.Sprintf(s.teamsUrl))
```

Comment:
Maybe we can add a check here that checks if the `teamsUrl` is set, otherwise we'll continue using `response, err := s.httpGet(client, fmt.Sprintf(s.apiUrl+"/teams"))`?

Reply:
Done